scripts/qa_crowbarsetup.sh

```diff
@@ -3149,12 +3149,25 @@ function get_ceph_nodes()
     fi
 }
 
+function manila_service_instance_get_uuid()
+{
+    local vm_uuid=`openstack --os-project-name manila-service server show manila-service -f value -c id`
+    test -n "$vm_uuid" || complain 91 "uuid from manila-service instance not available"
+    echo $vm_uuid
+}
+
+function manila_service_instance_get_floating_ip()
+{
+    local vm_uuid=`manila_service_instance_get_uuid`
```

Comment:
it works. See other examples in the script or i.e. http://www.linuxjournal.com/content/return-values-bash-functions